view/theme/frio/js/theme.js

```diff
@@ -784,8 +784,8 @@ function doActivityItemAction(ident, verb, un) {
 		// if no wait-rotator for activity(verb) is added, add it. or just show it, if exists
 		if ($('img[id^=waitfor-' + verb + '-' + ident.toString() + ']').length == 0) {
 			// append rotator to the shareMenu-button for small media
-			$('<img id="waitfor-' + verb + '-' + ident.toString() + '">')
-				.attr('src', 'images/rotator.gif')
+      $('<img>')
+        .attr('id', 'waitfor-' + verb + '-' + ident.toString())
 				.addClass('fa')
 				.appendTo($('button[id^=shareMenuOptions-' + ident.toString() + '] i:first-child' )).show();
```

Comment:
Is the `show()` call even necessary?
```suggestion
				.appendTo($('button[id^=shareMenuOptions-' + ident.toString() + '] i:first-child' ))
				.show();
```